src/panels/config/devices/device-detail/integration-elements/mqtt/mqtt-messages.ts

```diff
@@ -26,6 +25,8 @@ class MQTTMessages extends LitElement {
 
   @state() private _showTopic = false;
 
+  direction: string = "<direction>"
```

Comment:
Is this the right way to do it? Can a class attribute be abstract, so something like:
```ts
abstract readonly direction: string;
```